website/src/css/custom.css

```diff
@@ -45,14 +52,23 @@
   display: flex;
 }
 
-/* Make links more obvious */
+/* Make links more obvious [for light theme] */
 a {
   color: blue;
 }
 a:hover {
   color: blue;
 }
 
+/* Use default link color for dark theme */
+[data-theme='dark'] a {
+  color: var(--ifm-color-primary);
+}
+
+[data-theme='dark'] a:hover {
+  color: var(--ifm-color-primary);
+}
```

Comment:
Possibly out of scope, but there was a design fad for probably the last decade or so where people were removing the underlines from links for aesthetic reasons. I think we are probably starting to move back away from that because it impairs accessibility, and even for people who are fully sighted, it is sometimes difficult to identify links if the colors are too similar to the normal color. You might consider adding the underlines back for both light and dark themes (`text-decoration: underline`).

To make things look a bit prettier even with underlines, you could consider using some relatively new CSS properties for controlling the [offset](https://developer.mozilla.org/en-US/docs/Web/CSS/text-underline-offset) and [thickness](https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration-thickness) of the underlines. (Note that older browsers, particularly Safari, [might not support these](https://caniuse.com/text-decoration), but it's still safe to put in the CSS properties because they degrade gracefully for browsers that don't support them.)

Reply:
I agree with this idea but want to do it as a separate change.

I have tried it out in the past, and it's less about the ugliness (although I do appreciate the suggestions for slight prettification, will definitely keep in mind). The main problem is the straightforward way to do it adds underlines _everywhere_, even to headers, sidebars, footers. I would like to apply the change but be a little more intentional with it. Thus, it just is more work than I'm intending for this particular PR.